docs/self-managed/identity/user-guide/configuration/configure-logging.md

```diff
@@ -0,0 +1,116 @@
+---
+id: configure-logging
+title: "Configure logging"
+sidebar_label: "Configure logging"
+description: "In this guide we will demonstrate how to configure logging in the Identity component"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Configuring logging
+
+The Identity component uses the [Log4j2](https://logging.apache.org/log4j/2.x/) framework to control
+the log level and log format.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The included configuration file here is missing any description. We should add something like "following is the log4j configuration included in Identity".